### DISCUSSION TITLE
wire:ignore disable input validation

### DISCUSSION THREAD

**Discussion by windsondias** [livewire/livewire#3090] [7. Bugs (archived)]
### Description

When I use `wire:ignore` with input for example it stays disabled after validation is complete. 

This issue #1135 was fixed in that pull #1166 apparently in version 1x.

In this case, I'm using `onchange="@this.set('name', this.value)"` because the input is with the inputmask plugin

### Exact steps to reproduce

```
<form wire:submit.prevent="save">
    <input wire:ignore onchange="@this.set('name', this.value)" type="text"
               class="form-control @error('name') is-invalid @enderror"
               id="name" name="name">
        @error('name')
        <span class="invalid-feedback" role="alert">
            {{ $message }}
        </span>
        @enderror
    <button type="submit">Save</button>
</form>
```

`public function save() {
    $this->validate([
        'name' => 'required',
    ]);
}`

if I call a save method (`wire:submit.prevent)` with validation (if title is empty) it diabled all form elements and enabled them except the ones inside wire:ignore stays disabled

### Stripped-down, copy-pastable code snippets

**Context**
- Livewire version: [e.g. 2.2.3]
- Laravel version: [e.g. 8.3.1]
- Alpine version: [e.g. 2.3.0]
- Browser: [e.g. Chrome, Safari]

**Comment by mahmoudelshenawy** [comment]
can any one help me with this issue
The whole form is dispearing once i start typing in any of them in livewire model bind , i have tried using wire:ignore and it stops that behavior but the validation errors did not show

**Comment by austenc** [comment]
What is the purpose of using `wire:ignore` here? Typically that is used to prevent Livewire from modifying DOM, such as when working with other third-party javascript libraries. In the above example, I'd recommend `wire:model` instead of `wire:ignore` and `onchange`.

**Comment by mahmoudelshenawy** [comment]
if i dont use wire:ignore one i start typing in the field it disapears , if it is there it won't disapear but on the other side wire:ignore prevent displaying the error messages from validation,  i'm stuck in this case
plus i am using wire:model

**Comment by austenc** [comment]
@mahmoudelshenawy 

Please provide a public repo or more code to help us reproduce the issue. Livewire shouldn't cause fields to disappear, so it must be something with your setup. Are you using any extra javascript libraries that might be conflicting?

**Comment by mahmoudelshenawy** [comment]
yeah this is what i think also i am using a bootstrap theme with many plugins i dont know what exactly causing the conflict ,
but thank you for your time

**Comment by windsondias** [comment]
@austenc 

I created a repository to illustrate more clearly the reason for using wire: ignore. I created three inputs in different ways to show why just ignore works. To test if downloading.

@mahmoudelshenawy 

If you wish, you can add your code in this sample repository to demonstrate your problem.

**Comment by joshhanley** [comment]
@windsondias thanks for making an example repo! Are you able to share the link to it? And what to do when running the example (ie what route is the issue shown on/ do we have to click or input anything/ etc).

**Comment by windsondias** [comment]
@joshhanley 

Hello, I can't share a link, but if you go up locally it will open on the main route and with the descriptions of each input created.

**Comment by joshhanley** [comment]
@windsondias I'm sorry, but you need to at least share the link to your Github repo otherwise we can't investigate the issue for you. Please share it here so someone can have a look 🙂

**Comment by windsondias** [comment]
@joshhanley 

I'm sorry, I thought I added the repository link in the message I quoted him. I thought you were asking for a repository link running online.

Here is a link to the repo:
https://github.com/windsondias/livewire-exemple

**Comment by joshhanley** [comment]
@windsondias ah right. No worries! If I get time later I'll have a quick look at it.

**Comment by usamamuneerchaudhary** [comment]
Any update on this issue yet? Facing same kindof error.
